virtool/postgres.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import sys
+
+import sqlalchemy
+from sqlalchemy import create_engine
+
+logger = logging.getLogger(__name__)
+
+
+async def connect(postgres_connection_string: str) -> sqlalchemy.engine:
+    if not postgres_connection_string.startswith("postgresql://"):
+        logger.fatal("Invalid PostgreSQL connection string")
+        sys.exit(1)
+
+    try:
+        postgres = create_engine(postgres_connection_string)
+        connection = postgres.connect()
+
+        await check_version(connection)
+
+        return connection
+    except ConnectionRefusedError:
+        logger.fatal("Could not connect to PostgreSQL: Connection refused")
+        sys.exit(1)
+
+
+async def check_version(connection):
+    with connection as con:
+        info = con.execute('SHOW server_version').fetchone()
```

Comment:
This is not async.

This version of `sqlalchemy` is supposed to have asyncio support. Can you check into this please?